### DISCUSSION TITLE
How do I show full path of file below statusline?

### DISCUSSION THREAD

**Discussion by jjaimealeman** [NvChad/NvChad#619] [Q&A] [answered]
I'm a new user of NvChad and really liking it.

I have a bunch of `readme.md` files and jump around quite a bit. By default, it shows the `folder/readme.md` in the tabs at the top. It just makes the tabs really wide after I have a bunch of them open.

I would like to have the full path shown in the statusline instead.

How can I do this?

Here is what the tabs look like

https://i.imgur.com/zL9o1DP.png

**Comment by siduck** [comment, marked as answer]
oops sorry for late response, you can replace the filename component and write your own function for this.

**Comment by RenovZ** [comment]
@siduck Sorry to trouble you, could you please show me the code to support this feature, I also need this. Thanks very much.

**Reply by siduck** [reply]
hi @cgxxv read from here https://nvchad.com/config/nvchad_ui

**Reply by DougAnderson444** [reply]
```lua
-- chadrc.lua
local M = {}

M.ui = {
  statusline = {
    overriden_modules = function(modules)
      table.insert(
        modules,
        2,
        (function()
          local path = vim.api.nvim_buf_get_name(0):match "^.*/"
          return "%#St_LspStatus#" .. path -- https://github.com/NvChad/base46/blob/v2.0/lua/base46/integrations/statusline.lua
        end)()
      )
    end,
  },
}
return M
```
